packages/source-iottwinmaker/src/property-value/client/getPropertyValueByEntity.ts

```diff
@@ -0,0 +1,60 @@
+import { GetPropertyValueCommand, GetPropertyValueResponse, IoTTwinMakerClient } from '@aws-sdk/client-iottwinmaker';
+import { DataType, DataStreamBase } from '@iot-app-kit/core';
+import { TwinMakerStaticDataQuery } from '../types';
+import { toValue } from '../../utils/propertyValueUtils';
+import { isNumber } from 'lodash';
+
+export const getPropertyValueByEntity = async ({
```

Comment:
add a one liner what is this function is about?

Reply:
ok